scripts/README.md

```
@@ -0,0 +1,381 @@
+are converted to dashes to match the filename convention.
+
+If multiple files match (e.g. the same model was re-run against a newer Vera
+compiler), the most recently modified file wins. The Vera compiler version
+displayed in the chart subtitle is auto-detected from the filenames of the
+Vera full-spec results.
+
+### Missing-file behaviour
+
+If a file is missing, the script prints a warning like
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)